app/models.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import unicode_literals
+
+import os
```

Comment:
I tried this but Python insists (through errors) that `from __future__ import unicode_literals` should always be the first line in a file